resources/generate-money-factory.php

```diff
@@ -49,15 +48,21 @@ public static function __callStatic(string $method, array $arguments): Money
 
     $methodBuffer = '';
 
-    $currencies = iterator_to_array(new Currencies\AggregateCurrencies([
+    $iterator = new Currencies\AggregateCurrencies([
         new Currencies\ISOCurrencies(),
         new Currencies\BitcoinCurrencies(),
```

Comment:
Maybe BitcoinCurrencies should be deprecated/removed since the CryptoCurrencies class already handles BTC

Reply:
I agree with deprecating BitcoinCurrencies.

As for XBT, I would suggest to discuss here:

https://github.com/moneyphp/crypto-currencies/issues/2

Reply:
@frederikbosch BitcoinCurrencies will be integrated inside CryptoCurrencies so the feature will be duplicated. 

Marking it as deprecated will help developers to move to the new CryptoCurrencies without adding any breaking changes. 

Do you agree on deprecating it on the next major version at least?

Reply:
@rogervila the code `XBT` is not included in `CryptoCurrencies`, as it is included with the code `BTC`. Hence, I would like to leave it in. But even if it would contain `XBT`, deprecation suggests future removal, and I would only want to disturb people with refactoring their code that depends on this library if we have a technical reason for it. And I believe that reason is not there.

Reply:
Ok. Approving the PR.